sdk/deviceupdate/Azure.Iot.DeviceUpdate/src/UpdatesClient.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Iot.DeviceUpdate.Models;
+
+namespace Azure.Iot.DeviceUpdate
+{
+    /// <summary>
+    /// Update management service client.
+    /// </summary>
+    public partial class UpdatesClient
```

Comment:
You typically need only one ClientOptions per library.

Reply:
Yeah; this could be updated. Correct.